### ISSUE TITLE
naming of to_*_hull_range_from_latlon() unfortunate.

### ISSUE THREAD

**Issue by NiklasPhabian** [SpatioTemporal/pystare#49]
We have two functions:

```python
to_hull_range_from_latlon(lat, lon, level)
to_nonconvex_hull_range_from_latlon(lat, lon, level)
``` 

Their naming is a bit unfortunate.

From the user's perspective what is really happening is that we have two functions that take a set of lat/lon pairs and interpret them as a ring (!).  ```to_hull_range_from_latlon()``` then converts the ring into a convex hull (the convex hull, of course, is also a ring; though the ordering is only relevant for the ring, not for the hull). Then both functions tesselate the ring with trixels and return the spatial range holding the SIDs of all those trixels. 
This is of course a bit different implementation-wise. But the user really shouldn't be bothered with that.

I think the functions should thus be renamed to

```python
range_from_hull(lat, lon, level)
range_from_ring(lat, lon, level,)
```

**Comment by NiklasPhabian** [comment]
Other options

```python
hull_to_range()
ring_to_range()
```

or

``` python
hull_to_cover()
ring_to_cover()
```

**Comment by NiklasPhabian** [comment]
I am going to add new aliases to the functions and throw Deprecation warnings for 

```python
to_hull_range_from_latlon(lat, lon, level)
to_nonconvex_hull_range_from_latlon(lat, lon, level)
```

**Comment by NiklasPhabian** [comment]
```python
cover_from_*()
``` 
probably makes the most sense since we already introduced the word "cover" in e.g.. ```to_circular_cover()```

**Comment by NiklasPhabian** [comment]
closed with #61
